src/docs/version-conf.js

```diff
@@ -0,0 +1 @@
+conf = { version: '7.6.0-test'}
```

Comment:
This string will be only until the release. It will be updated with proper value automatically. But I will commit '-SNAPSHOT' now.